walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,134 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    Collection<TimeRange> return_value;
+
+    List<TimeRange> listtransfer = new ArrayList<TimeRange>();
+    Event[] event_array = events.toArray(new Event[events.size()]);
+
+    int request_duration = (int)request.getDuration();
+
+    
+
+
+    if(events.size() == 1){
```

Comment:
Not quite sure why you need all these cases, could you add some comments explaining the branches?